### PR TITLE
0.5

### PR DESCRIPTION
**Pull request by ddavydenko**
### Description of changes:
Pulling in changes from 0.5 into master (project description for pypi publishing is the only new thing, everything else is already in master)

### CODE REVIEW

File: smdebug/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.5.0"
+__version__ = "0.5.0.post0"
```
Comment (anirudhacharya) [inline smdebug/_version.py]: nit: new line

why this change? in the nightly pre-build release, we fetch this version number append 'b' 'timestamp' to generate the pre-build version. this upload upload might now fail since `0.5.0.post0b<timestamp>` might not meet semver standards of pypi.

File: setup.py [hunk 2]
```diff
@@ -25,6 +25,7 @@
 import smdebug
 
 DOCLINES = (__doc__ or "").split("\n")
+CURRENT_VERSION = __version__
```
Comment (anirudhacharya) [inline setup.py]: are we using this anywhere?

File: smdebug/_version.py (hunk 1 above)
Comment (ddavydenko) [reply smdebug/_version.py]: this was because 0.5.0 publish happened without description for pypi so next version after that is 0.5.0.post0.

File: setup.py (hunk 2 above)
Comment (ddavydenko) [reply setup.py]: not sure... I was under impression that is used but if it is not - happy to remove it.
